services/repository/src/main/java/org/sagebionetworks/repo/web/service/EvaluationServiceImpl.java

```diff
@@ -170,10 +174,17 @@ public Submission createSubmission(Long userId, Submission submission, String en
 		Long versionNumber = submission.getVersionNumber();
 		EntityBundle bundle = serviceProvider.getEntityBundleService().getEntityBundle(userId, entityId, versionNumber, mask);
 		Submission created = submissionManager.createSubmission(userInfo, submission, entityEtag, submissionEligibilityHash, bundle);
-		List<MessageToUserAndBody> messages = submissionManager.
-				createSubmissionNotifications(userInfo,created,submissionEligibilityHash,
-						challengeEndpoint, notificationUnsubscribeEndpoint);
-		notificationManager.sendNotifications(userInfo, messages);
+		
+		try {
```

Comment:
We agree this is a "hack" but is needed because the "real" fix is too risky to put in staging.